crrExistingObjects.js

```diff
@@ -17,11 +17,13 @@ let TARGET_REPLICATION_STATUS = process.env.TARGET_REPLICATION_STATUS;
 const WORKERS = (process.env.WORKERS &&
                  Number.parseInt(process.env.WORKERS, 10)) || 10;
 const MAX_UPDATES = (process.env.MAX_UPDATES &&
-                     Number.parseInt(process.env.MAX_UPDATES, 10));
+                     Number.parseInt(process.env.MAX_UPDATES, 10)) || 10;
```

Comment:
MAX_UPDATES should not have a default value not to limit the number of updates by default, it's on purpose :wink: